Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -0,0 +1,176 @@
+<textarea class="not-handwritable"></textarea>                                         <!-- the computed handwriting value is false -->
+
+<textarea></textarea>                                         <!-- the computed handwriting value is true -->
+<textarea style="touch-action:handwriting;"></textarea>       <!-- the computed handwriting value is true -->
+<textarea style="touch-action:pan-x;"></textarea>             <!-- the computed handwriting value is false -->
+<textarea style="touch-action:pan-x handwriting;"></textarea> <!-- the computed handwriting value is true -->
+<textarea style="touch-action:manipulation;"></textarea>      <!-- the computed handwriting value is true -->
+
+
+<div style="touch-action:pan-x;">
+  <textarea></textarea>                                       <!-- the computed handwriting value is false -->
+  <textarea style="touch-action:handwriting;"></textarea>     <!-- the computed handwriting value is true -->
+  <textarea style="touch-action:pan-y;"></textarea>           <!-- the computed handwriting value is false -->
+</div>
+
+<div class="not-handwritable">
+  <textarea></textarea>                                       <!-- the computed handwriting value is false -->
+  <textarea class="handwritable"></textarea>     <!-- the computed handwriting value is true -->
+</div>
+```
+
+### Keywords and States
+
+The `handwriting` keyword indicates whether an element and the element's descendants will allow handwriting input when supported by the user agent. Handwriting will only be allowed for an element when its computed `touch-action` includes the `handwriting` keyword. By default, `auto` and `manipulation` will include the `handwriting` keyword.
+
+#### Determining enablement
+
+All CSS properties have computed values for all elements. The enablement of handwriting in a given `element` can be determined by running the following steps:
+
+1. If the computed value for `touch-action` on `element` does not contain the `handwriting` or `manipulation` keyword, **disable handwriting**.
+2. If the computed value for `touch-action` on `element` does contain the `handwriting` or `manipulation` keyword, **enable handwriting**.
+3. If the computed value for `touch-action` on `element` is `auto`, search `element`'s parent chain for an ancestor with a non-`auto` computed value for `touch-action`. Apply steps 1 and 2 to the computed value for `touch-action` on the lowest such ancestor.
+4. If the computed value for `touch-action` on `element` and all of its ancestors is `auto`, **enable handwriting**.
+
+### Caveats
+
+A few pain points have been brought up that are worth discussion:
+* Web pages who currently have the `touch-action` property set for different Elements will lose the handwriting capabilities on this element even if they don't want to disable it. When the new keyword ships, the absence of the value will be interpreted as the author of the webpage intently disabling handwriting.
```

Comment:
I think 'who' should be replace with 'that' and uppercase Elements changed to lowercase.

Reply:
There were many "who"s that should've been "that"s, fixed, thanks.